src/txn/txn_rollback_to_stable.c

```diff
@@ -646,32 +646,54 @@ __rollback_abort_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_COL *cip, W
          */
         WT_RET(__wt_page_cell_data_ref(session, page, vpack, &buf));
 
-        WT_ERR(__wt_upd_alloc(session, &buf, WT_UPDATE_STANDARD, &upd, NULL));
         /*
-         * Set the transaction id of updates to WT_TXN_NONE when called from recovery, because the
-         * connections write generation will be initialized after rollback to stable and the updates
-         * in the cache will be problematic. The transaction id of pages which are in disk will be
-         * automatically reset as part of unpacking cell when loaded to cache.
+         * For prepared transactions, it is possible that both the on-disk key start and stop time
+         * windows can be same. To abort these updates, check for any stable update from history
```

Comment:
on-disk key's start and stop time point can be the same

Reply:
Fixed.